x-pack/test/functional/apps/ml/aiops/index.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, loadTestFile }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  describe('machine learning - aiops', function () {
+    this.tags(['skipFirefox', 'ml', 'walterra']);
```

Comment:
Think this walterra was left in accidentally 😄

Reply:
Oh yes 😃 ! Fixed in [5b14409](https://github.com/elastic/kibana/pull/138387/commits/5b1440910fdebf4a0178b90a1aa7dc9cbbf179c2).